rice-apps/Carpool-V3
# Add join/leave email notifications

# Description

Notify the ride owner when someone leaves or joins their ride.

Note: the signature image in the html version of the email won't currently show up because it depends on this PR being merged into production.

## Type of change


# How Has This Been Tested?

You'll have to add `SENDER_EMAIL=ricecarpool@gmail.com` and `SENDER_EMAIL_PASSWORD` to the `api/` `.env`. You can get the password from Shreya.

- Join someone's ride (or have them join yours)
- Check the owner's inbox for an email
- Make sure it doesn't say or link to anything wacky

Spec from Shreya:

```
subject: Rice Carpool: Shreyas left your ride
text:
Shreyas Minocha left your ride from (Rice) to (IAH) on (May 6, 2022).
See (link) for details.
[rice carpool email signature]
```
